.github/workflows/on-pull-request.yml

```diff
@@ -50,6 +50,24 @@ jobs:
       - run: npm run build
         working-directory: frontend
 
+      - run: npm run start &>/dev/null &
```

Comment:
This will not serve the build created in the previous step right?

Reply:
No it won't.
I did that earlier, but then it would try to use `https://acc.looplijst.top.amsterdam.nl/api/v1/` as its backend, which is not what I wanted. (We can't bypass login there)